chucks-pick3-web/angular-app/src/app/models/customer-info.model.ts

```diff
@@ -0,0 +1,19 @@
+import {CustomerInfo} from './customer-info';
+
+export class CustomerInfoModel implements CustomerInfo {
+  public firstName: string;
+  public lastName: string;
+  public email: string;
+  public phone: string;
+  public city: string;
+  public state: string;
+
+  constructor(config: CustomerInfo) {
+    this.firstName = config.firstName;
+    this.lastName = config.lastName;
```

Comment:
You need to double check and make sure `config` is not null and not undefined before you can access the `config.lastName` property.

Reply:
Checked to see if config is not null or undefined.